db/models/order.js

```diff
@@ -0,0 +1,21 @@
+'use strict'; // eslint-disable-line semi
+/* eslint-disable camelcase */
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Order = db.define('orders', {
+  status: {
+    type: Sequelize.ENUM('Created', 'Processing', 'Cancelled', 'Completed'),
+    defaultValue: 'Created'
+  },
+  isActive: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: true
+  },
+  items: {
```

Comment:
//TODO: getter method to take items and turn into an array of objects with product.id and product.quantity